CHANGELOG.md

```diff
@@ -80,3 +84,6 @@ Fixes:
 Added support for pretty printers in variable list (by @gentoo90), enable
 with `"valuesFormatting": "prettyPrinters"` if you have a pretty printer
 to get potentially improved display of variables.
+
+[Unreleased]: https://github.com/WebFreak001/code-debug/compare/v0.26.0...HEAD
+[0.26.0]: https://github.com/WebFreak001/code-debug/compare/v0.25.1...v0.26.0
```

Comment:
I like adding this here, but having the links at the bottom of the file _will_ cause contributors to not update this every time a new release gets added. I would recommend moving this immediately after the version headers.

Reply:
I was looking for more structure in the Changelog and ran across the [Keep a Changelog](https://keepachangelog.com/) and thought it had a lot of good ideas.  I especially like the subsections where you group like changes (fixed, added, deprecated, etc).  I didn't convert the changelog to this format, as I wanted to get your buy-in, but I did add a couple ideas from it.  Notably, I started using the subsections to classify things and did add the "compare" links to the bottom.

If you like this approach, we can add a pre-release check to make sure the links are updated as expected prior to the release (possibly with [this github action](https://github.com/mindsers/changelog-reader-action)), similar to the current checks which make sure all of the release versions are synchronized.  Another option would be to automate part of this (possibly with [this github action](https://github.com/thomaseizinger/keep-a-changelog-new-release)).  I think there are few different ways we could address this concern if we want to keep moving in this direction.

Reply:
I'm fine with adding it, but the markdown link references at the bottom should really be grouped with the headers. For users viewing the README file without a markdown viewer it'd become much nicer to follow.

These indirect markdown links can be placed anywhere in the file I think, so we can either just put them immediately after the version header or put it at the end of each version block, which kinda makes more sense when reading it as well.